doc/installation.rst

```diff
@@ -3,26 +3,74 @@
 Installation
 ============
 
-splot supports python `3.5`_ and `3.6`_ only. Please make sure that you are
-operating in a python 3 environment.
+Installing dependencies
+-----------------------
 
-Installing released version
----------------------------
+`splot` is compatible with Python `3.6`_ and `3.7`_ and
+depends on GeoPandas 0.4.0 or later and matplotlib 2.2.2 or later.
+Please make sure that you are operating in a Python 3 environment.
 
-giddy is available on the TBA. Therefore, you can either
-install directly with `pip` from the command line::
+splot also uses
 
-  pip install splot
+* numpy
+* seaborn
+* mapclassify
+* Ipywidgets
 
+Depending on your spatial analysis workflow and the PySAL objects
+you would like to visualize, splot relies on:
 
-or download the source distribution (.tar.gz) and decompress it to your selected
-destination. Open a command shell and navigate to the decompressed folder.
-Type::
+PySAL >=2.0
 
-  pip install .
+or the installation of separate packages found in the PySAL stack:
 
-Installing development version
-------------------------------
+* esda
+* libpysal
+* spreg
+* giddy
+
+
+Installing the newest release
+-----------------------------
+
+There are two ways of accessing splot. First, splot is installed with
+the PySAL 2.0 metapackage through:
+
+```$ pip install -U pysal```
+
+or 
+
+```$ conda install -c conda-forge pysal```
+
+Second, splot can be installed as a separate package. If you are
+using Anaconda, install splot via the conda utility:
+
+```$ conda install -c conda-forge splot```
+
+Otherwise, you can install splot from PyPI with pip:
+
+```$ pip install splot```
+
+
+Troubleshooting
+---------------
+Most common installation errors are due to splot's dependency on GeoPandas.
+
+It often helps to first install GeoPandas separately from conda-forge with:
+
+```$ conda install --channel conda-forge geopandas```
+
+before installing splot (preferably also from conda, alternatively from pip).
+
+For more information on troubleshooting the installation of GeoPandas with pip, see the `GeoPandas`_ docuemntation.
+
+It is also possible to install splot with a later Python version (>3.7)
+through the separate installation of GeoPandas or through installation with conda-forge.
+(Note that splot is currently only tested for Python version 3.6 and 3.7)
```

Comment:
Should this be on a new line or a continuation of the previous text block?

Reply:
Great catching! I formatted the entire document to comply to pep8, so there are a couple line breaks. It is intended.